libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -64,12 +64,23 @@
         "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because measure value filters are configured for the pivot.",
         "limit": 0
     },
+    "visualizations.totals.dropdown.tooltip.nat.disabled.mvf|insight|metric": {
+        "value": "Rollup (Total) aggregation is not available when filtering the insight by metric value. To use Rollup (Total), remove all metric value filters from the filter bar.",
+        "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because metric value filters are configured for the pivot.",
+        "limit": 0
+    },
     "visualizations.totals.dropdown.tooltip.nat.disabled.mvf|report": {
         "value": "Rollup (Total) aggregation is not available when filtering the report by measure value. To use Rollup (Total), remove all measure value filters from the filter bar.",
         "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because measure value filters are configured for the pivot.",
         "limit": 0,
         "translate": false
     },
+    "visualizations.totals.dropdown.tooltip.nat.disabled.mvf|report|metric": {
+        "value": "Rollup (Total) aggregation is not available when filtering the report by metric value. To use Rollup (Total), remove all metric value filters from the filter bar.",
+        "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because measure value filters are configured for the pivot.",
```

Comment:
`measure` in the `"comment"` should be changed to `metric` as well, shouldn't it?

Reply:
fixed